### DISCUSSION TITLE
Required numpy version

### DISCUSSION THREAD

**Discussion by aslamahrahman** [shner-elmo/TradingView-Screener#38] [Q&A]
Looks like there is a required numpy version (<2) that the repo needs? I upgraded to numpy==2.0.0 and the package cracked.

Could someone confirm? Thanks!

**Comment by shner-elmo** [comment]
That makes sense, I will fix this soon

**Comment by shner-elmo** [comment]
@aslamahrahman completely forgot about this, can you try installing the latest (pre-release) version:
```
pip install tradingview-screener==3.0.0rc1
```
if everything works I'll publish it as `v3.0.0`
